tools/boards.txt.py

```diff
@@ -914,20 +914,18 @@
     ####################### menu.resetmethod
 
     'resetmethod_menu': collections.OrderedDict([
-        ( '.menu.ResetMethod.ck', 'ck' ),
-        ( '.menu.ResetMethod.ck.upload.resetmethod', 'ck' ),
-        ( '.menu.ResetMethod.nodemcu', 'nodemcu' ),
-        ( '.menu.ResetMethod.nodemcu.upload.resetmethod', 'nodemcu' ),
+        ( '.menu.ResetMethod.dtr', 'dtr' ),
+        ( '.menu.ResetMethod.dtr.upload.resetmethod', '--before default_reset --after hard_reset' ),
+        ( '.menu.ResetMethod.nodtr', 'no dtr' ),
```

Comment:
I would mention the old names with something like this:
* `dtr (aka nodemcu)`
* `no dtr (aka ck)`

so users are not lost

Reply:
Good idea, added changes.